plugins/notifications-backend/src/service/types.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { TokenManager } from '@backstage/backend-common';
+import { CatalogClient } from '@backstage/catalog-client';
+import { Config } from '@backstage/config';
+import { IdentityApi } from '@backstage/plugin-auth-node';
+import { PermissionEvaluator } from '@backstage/plugin-permission-common';
+
+import { Logger } from 'winston';
+
+export interface RouterOptions {
+  logger: Logger;
```

Comment:
Should be changed to `LoggerService` and remove the `winston` dependency

Reply:
I see winston to be used in other plugins as well at this point.

Reply:
Yeah at this point but backstage is moving all the time to the new backend system and I think it makes sense to build all new plugins to follow that.

Reply:
And related to the new backend system, you probably should add also the `src/plugin.ts` for the backend plugin as well

Reply:
done